app/assets/javascripts/dashboard/views/dash/dashboard_task_list_view.js

```diff
@@ -0,0 +1,317 @@
+                <li>
+                  <a href="#" onClick={() => this.cancelAnnotation(annotation.id)}>
+                    <i className="fa fa-trash-o" />
+                    cancel
+                  </a>
+                </li>
+              </div>
+            : <li>
+                <a href="#" onClick={() => this.finish(task)}>
+                  <i className="fa fa-check-circle-o" />
+                  finish
+                </a>
+              </li>}
+        </ul>;
+  };
+
+  resetTask(annotationId: string) {
+    const url = `/annotations/Task/${annotationId}/reset`;
+
+    Request.receiveJSON(url).then(jsonData => {
+      Toast.message(jsonData.messages);
+    });
+  }
+
+  cancelAnnotation(annotationId: string) {
+    const wasFinished = this.state.showFinishedTasks;
+    if (confirm("Do you really want to cancel this annotation?")) {
+      Request.triggerRequest(`/annotations/Task/${annotationId}`, { method: "DELETE" }).then(() => {
+        if (wasFinished) {
+          this.setState({
+            finishedTasks: this.state.finishedTasks.filter(t => t.annotation.id !== annotationId),
+          });
+        } else {
+          this.setState({
+            unfinishedTasks: this.state.unfinishedTasks.filter(
+              t => t.annotation.id !== annotationId,
+            ),
+          });
+        }
+      });
+    }
+  }
+
+  async getNewTask() {
+    if (this.state.unfinishedTasks.length === 0 || confirm("Do you really want another task?")) {
+      this.setState({ isLoading: true });
+      try {
+        const newTaskAnnotation = await Request.receiveJSON("/user/tasks/request");
+
+        this.setState({
+          unfinishedTasks: this.state.unfinishedTasks.concat([
+            convertAnnotationToTaskWithAnnotationType(newTaskAnnotation),
+          ]),
+        });
+      } finally {
+        this.setState({ isLoading: false });
+      }
+    }
+  }
+
+  handleTransferredTask() {
+    this.setState({ isTransferModalVisible: false });
+
+    const removeTransferredTask = tasks =>
+      tasks.filter(t => t.annotation.id !== this.state.currentAnnotationId);
+
+    if (this.state.showFinishedTasks) {
+      this.setState({
+        finishedTasks: removeTransferredTask(this.state.finishedTasks),
+      });
+    } else {
+      this.setState({
+        unfinishedTasks: removeTransferredTask(this.state.unfinishedTasks),
+      });
+    }
+  }
+
+  getCurrentTasks() {
+    return this.state.showFinishedTasks ? this.state.finishedTasks : this.state.unfinishedTasks;
+  }
+
+  renderTable() {
+    return (
+      <Table
+        dataSource={this.getCurrentTasks().filter(
+          task => task.annotation.state.isFinished === this.state.showFinishedTasks,
+        )}
+        rowKey="name"
+        pagination={{
+          defaultPageSize: 50,
+        }}
+      >
+        <Column
+          title="#"
+          dataIndex="id"
+          render={(__, task) => FormatUtils.formatHash(task.id)}
+          sorter
```

Comment:
Fixed it, I interpreted the antd documentation ("If you need sort buttons only, set it true") so that it would use the rendered value automatically for sorting. Wrong interpretation :smile: